### DISCUSSION TITLE
Doge Wallet won't rescan on BTC Pay Server

### DISCUSSION THREAD

**Discussion by jtmoran11** [btcpayserver/btcpayserver#6339] [Technical Support]
Our initial BTC Pay Server host disappeared, literally, site went down, stopped answering phones, emails bounced.  We had setup a native wallet, we didn't import some other wallet.  So scroll forward, we signed up with Luna Node and kicked off a BTC Pay Server instance on a new vm.  We setup Doge and entered our seed code to recover our previous native BTC Pay Server Doge wallet.  It recovered successfully.  The balance was off so we next tried to rescan.

We received the following message:

This feature is disabled

 The full node is synched (with a green check mark)
 You are server administrator (with a green check mark)
 This full node does not support rescan of the UTXO set (WITH A BIG RED X)

Any recommendations, account key path? Open the wallet elsewhere, transfer everything out and start a fresh one?

**Comment by pavlenex** [comment]
> Open the wallet elsewhere, transfer everything out and start a fresh one?

Probably the best thing to do is this. As to which wallet should _elsewhere_  be I don't have an answer, you should try a dodge community and ask if there's a wallet with custom gap limit, perhaps there's Electrum fork for Dodge.
